DEFAULT_CONFIG.json5

```diff
@@ -354,11 +354,11 @@
       open_timeout: 10000,
       /// Timeout in milliseconds when accepting a link
       accept_timeout: 10000,
-      /// Maximum number of zenoh session in pending state while accepting
+      /// Maximum number of links in pending state while performing the handshake for accepting it
       accept_pending: 100,
-      /// Maximum number of sessions that can be simultaneously alive
+      /// Maximum number of transports that can be simultaneously alive for a single zenoh sessions
```

Comment:
I thought it was the maximum number of sessions for a single transport based on the config name `max_sessions`. Is it possible for us to change the name? For example, `max_per_session` or something else.

Reply:
That would introduce a breaking change in the config I would rather avoid to introduce.